libs/wingsdk/test/target-awscdk/__snapshots__/bucket.test.ts.snap

```diff
@@ -305,6 +305,7 @@ def submit_response(event: dict, context, response_status: str, error_message: s
           ],
         },
         "Runtime": "nodejs18.x",
+        "Timeout": 900,
```

Comment:
where are those coming from?

Reply:
I'm not doing anything, trust me 😂
I just ran `pnpm test` inside wingsdk and getting this snapshot.

Reply:
@MarkMcCulloh do you have any idea?

Reply:
My guess is that the lockfile was updated and the newer version of aws-cdk-lib added this default.